server.js

```diff
@@ -1,4 +1,4 @@
-import express from 'express';
+var express = require('express');
```

Comment:
@pheuter  thank you for reply, I understand the idea, but unfortunately without this change I was not able to run this on Heroku (and looks like I'm not the only one).

Please see your original app deployed with Heroku button here: https://original-essential-react.herokuapp.com/ (gives error)
And here is same deployment to Heroku with button from my forl (1 line changed): https://fix-import-essential-react.herokuapp.com/ (works as expected)